lms/static/sass/features/_course-experience.scss

```diff
@@ -171,3 +171,21 @@
     }
   }
 }
+
+// Course Updates Page
+div.course-updates {
+  div.all-updates {
+      article.updates-article {
+        margin: 10px 0px;
+        .date {
+          font-size: 14px;
+          font-weight: 300;
+        }
+        .end-line {
```

Comment:
I mentioned this in the code already, but we don't create elements just for styling.